app/errors.py

```diff
@@ -0,0 +1,22 @@
+class VaccineError(Exception):
+    """
+    parent vaccine error
+    """
+
+
+class NotVaccinatedError(VaccineError):
+    """
+    not vaccinated
+    """
+
+
+class OutdatedVaccineError(VaccineError):
+    """
+    outdated vaccine
+    """
+
+
+class NotWearingMaskError(Exception):
+    """
+    no mask
+    """
```

Comment:
All your custom exceptions are missing descriptive error messages. It would be better to add a custom message to each of them. This can be done by overriding the '__init__' method in each class and passing the custom message to the parent class's '__init__' method. For example: 'class VaccineError(Exception): def __init__(self, message="Vaccine error"): super().__init__(message)'.